docs/src/documentation/03-components/07-interaction/stepper/03-accessibility.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/stepper/accessibility/
+---
+
+## Accessibility
+
+### Stepper
+
+The Stepper component has been designed with accessibility in mind. It can be used with keyboard navigation and includes properties that enhance the experience for users of assistive technologies.
+
+Although the `ariaLabel` and `ariaLive` props are optional for the Stepper (StepperStateless) component itself, it is recommended to fill them in to ensure that the component can be perceived by assistive technologies. This ensures that the Stepper (StepperStateless) component is accessible.
+
+Also, it is highly recommended to fill the `titleDecrement` and `titleIncrement` props in to provide a description of the decrement and increment buttons for screen readers.
+
+#### Example
+
+The content of children component is text, so it is read by screen reader.
+
+```jsx
+<Stepper
```

Comment:
I would maybe add more elements to this example, as it probably wouldn't be placed in isolation but with some label? (This is based on the comment I put to the review with the image example, I just wanted to post it also here in case we add all these other props.)

Reply:
I added an example when using Stepper with connected title (`ariaLabelledBy` prop is used).